packages/style-kit/stories/field-elements.stories.scss

```diff
@@ -3,4 +3,21 @@
 :local(.story) {
   @include typography-main($meeco-theme, true);
   @include inputs-main($meeco-theme);
+
+  margin: $grid * 2;
+
+  max-width: 600px;
+
+  .container {
+    margin-top: $grid * 2;
+  }
+
+  .attachment {
+    margin-top: $grid * 2;
+    .icon {
```

Comment:
It's sort of expected this will always be here isn't it? In which case it probably belongs in the actual style kit code rather than the storybook code.